### DISCUSSION TITLE
remove underlines in strings

### DISCUSSION THREAD

**Discussion by aum7** [LunarVim/LunarVim#3799] [Q&A] [answered]
olo

in my .lua file, while a cursor is inside multi-line string, ie
```lua
[[
string 
string
string
]]
```
all text is underline, and it bothers me
also, when cursor is at any word,
anywhere in text, of same .lua file,
all same words are also underscored
i believe this is linter's work, maybe treesitter plugin's work

in config.lua file, there is default colorscheme
lvim.colorscheme = "lunar"
i just f***** love it !
so, i tried few other colorschemes, but finished back to lunar
and doing this, the annoying underlining disappeared - hurrai !
though, the change was not permanent
if i restart lvim, underlining comes back
so, i either do
:colorscheme lunar
every time i start lvim
or somebody helps me solve this issue to be permanent
i guess by entering some line into my config.lua file

please advise

godspeed

aum
s love nia
![lunarvim_underlines](https://user-images.githubusercontent.com/9611505/216053557-8055adef-448d-475a-9516-f52801a7ba94.png)

**Comment by LostInTheLogs** [comment, marked as answer]
disable the builtin illuminate plugin

**Reply by aum7** [reply]
thank you for your answer

like
:illuminate disable ?

**Reply by aum7** [reply]
i took a look, there is no 'illuminate' serch-word / setting in my config.lua
now what ? how do i disable illuminate / underlining ?

godspeed

**Reply by LostInTheLogs** [reply]
it's a builtin plugin, use `lvim.builtin.illuminate.active = false`

**Reply by aum7** [reply]
ok, thx, you da man !
the weird behavior, afaik, still remains -
if i do :colorscheme habamax (my default now),
the underlining goes away

have fun
